tests/Unit/Hooks/CheckoutScriptHooksTest.php

```diff
@@ -27,6 +27,7 @@ function (
         factory(CheckoutSettings::class)
             ->withEnableDeliveryOptions($enableDeliveryOptions)
             ->withEnableDeliveryOptionsWhenNotInStock($enableDeliveryOptionsWhenNotInStock)
+            ->withDeliveryOptionsPosition('woocommerce_after_checkout_billing_form')
```

Comment:
Dit komt nergens terug en is dus geen test of die waarde wel correct terugkomt ofzo